00-demo.md

```diff
@@ -0,0 +1,37 @@
+# Demo Chapter
+
+Here's some code that'll run in a notebook and also run in CI.
+
+```python
+print("Hello world")
+```
+
+## How to Use the Setup
+
+You can write chapters for the course using standard markdown files in a variety of ways. Here are a few approaches that we've found useful. Before getting started, you need to set up a Python environment and install the Python requirements. Once you have Python set up, run `pip install -r requirements.txt` from the root directory to install all the requirements.
+
+### The Bare-Bones Approach
+
+1. Use a text editor of your choice to modify the markdown. Any fenced code block starting with the `python` language identifier will be executed when you convert the markdown to html.
+2. Run `make [your markdown document].ipynb.html` from the command line to execute the notebook and convert it to html.
+
+### The Interactive Approach
+
+Recompiling the document with `make` every time you make a small change can be cumbersome. You can get an interactive environment using Jupyter Lab.
+
+1. Run `jupyter lab` from the command line to start a new Jupyter Lab server.
+2. Once the server has loaded (it can take a few seconds), right-click on the markdown file you want to modify, hover your mouse over "Open With", and select "Notebook".
```

Comment:
```suggestion
2. Once the server has loaded (it can take a few seconds), right-click on the markdown file you want to modify, hover your mouse over "Open With", and select "Notebook". If you don't see the "Notebook" option, shut down the server, execute `jupyter server extension enable jupytext`, then restart the server. 
```